### ISSUE TITLE
Request: Setup Docker Hub Auto Builds

### ISSUE THREAD

**Issue by tony1661** [perara/wg-manager#64]
Just wanna say I love the project!

I prefer to deploy with docker (docker hub more specifically). I've noticed the docker hub repo is a bit out of date and doesn't have the recent modifications.

Can we setup automatic builds from github if possible?

**Comment by perara** [comment]
Yes! This is an excellent idea. I'm hoping to do this in the upcoming days

**Comment by alexlii1971** [comment]
+!

**Comment by perara** [comment]
Upcoming days he said... Ive done abit work on this, but somehow it "crashes" during builds.

**Comment by perara** [comment]
track https://github.com/jina-ai/jina/issues/1695 for arm fix

**Comment by perara** [comment]
The auto build is working again. but only for amd64 builds
https://hub.docker.com/repository/docker/perara/wg-manager/tags?page=1&ordering=last_updated
